charts/wildfly-common/templates/_ingress.yaml

```diff
@@ -0,0 +1,43 @@
+{{- define "wildfly-common.ingress.tpl" -}}
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ include "wildfly-common.appName" . }}
+  labels: {}
+  annotations: {}
+spec:
+  {{- if .Values.deploy.ingress.class }}
+  ingressClassName: {{ .Values.deploy.ingress.class }}
+  {{- end }}
+  {{- if .Values.deploy.ingress.tls.enabled }}
+  tls:
+    - 
+       {{- if .Values.deploy.ingress.host }}
+      host: 
```

Comment:
should be `hosts` according to https://kubernetes.io/docs/concepts/services-networking/ingress/#tls